lib/OperatingSystems/FreeBSD.php

```diff
@@ -46,12 +46,12 @@ public function getMemory(): Memory {
 		}
 
 		$matches = [];
-		$pattern = '/(?>\/dev\/\w+)\s+(?>\d+)\s+(?<Used>\d+)\s+(?<Avail>\d+)\s+(?<Capacity>\d+)/';
+		$pattern = '/(?>\/dev\/\S+)\s+(?>\d+)\s+(?<Used>\d+)\s+(?<Avail>\d+)\s+(?<Capacity>\d+)/';
```

Comment:
Why the change ?

Reply:
/dev/mirror/swap4.eli
^the 2nd backslash blocks the match with regex when using \w+

Reply:
/dev/mirror/swap4.eli
^the 2nd backslash blocks the match with regex when using \w+